src/components/navigation-logo/navgation-logo.tsx

```diff
@@ -0,0 +1,73 @@
+import { Component, Element, h, Host, Prop, VNode } from "@stencil/core";
+import { CSS } from "./resources";
+
+@Component({
+  tag: "calcite-navigation-logo",
+  styleUrl: "navigation-logo.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+export class CalciteNavigationLogo {
+  //--------------------------------------------------------------------------
+  //
+  //  Element
+  //
+  //--------------------------------------------------------------------------
+  @Element() el: HTMLCalciteNavigationLogoElement;
+
+  //--------------------------------------------------------------------------
+  //
+  //  Public Properties
+  //
+  //--------------------------------------------------------------------------
+  /** When true, the component is highlighted. */
+  @Prop({ reflect: true }) active: boolean;
+
+  /** Specifies the URL destination of the component, which can be set as an absolute or relative path.*/
+  @Prop({ reflect: true }) href: string;
+
+  /** Describes the appearance or function of the `thumbnail`. If no label is provided, context will not be provided to assistive technologies. */
+  @Prop() label: string;
+
+  /** Specifies the subtext to display, such as an organization or application description. */
+  @Prop() subtext: string;
+
+  /** Specifies the text to display, such as a product name.*/
+  @Prop() text: string;
+
+  /** When `true`, displays the `text` and `subText` contents. */
+  @Prop({ reflect: true }) textEnabled = false;
+
+  /** Specifies the `src` to an image. */
+  @Prop() thumbnail: string;
+
+  // --------------------------------------------------------------------------
+  //
+  //  Render Methods
+  //
+  // --------------------------------------------------------------------------
+  render(): VNode {
+    return (
+      <Host>
+        <a href={this.href} tabIndex={0}>
```

Comment:
Is href an optional property? Was it considered that the user could just wrap the component in their own <a> so that the user can set other anchor attributes like rel, title, target, etc?

Reply:
In other places we accept href like Button, we also accept rel and target props, which we could add here. 

Asking to wrap a component in anchor seems like it would introduce a lot of potential style, focus, behavior fragmentation.

Reply:
yeah makes sense. 👍